pkgs/development/libraries/mesa/default.nix

```diff
@@ -5,9 +5,10 @@
 , llvmPackages, libffi, libomxil-bellagio, libva-minimal
 , libelf, libvdpau
 , libglvnd, libunwind
-, vulkan-loader
+, vulkan-loader, glslang
 , galliumDrivers ? ["auto"]
 , vulkanDrivers ? ["auto"]
+, videoCodecs ? [ "h264dec" "h264enc" "h265dec" "h265enc" "vc1dec" ]
```

Comment:
@jtojnar what do you think, list or `withPatentedCodecs ? true`?

Reply:
I would guess people would either want to disable or enable all encumbered codecs but I do not have a preference.

I worry about this list getting desynchronized. Especially when I did not see any mention of this change in the release notes, and was only aware of it because someone on GNOME IRC posted a link about Fedora dealing with this.

But I am not sure how to solve that other than asking upstream to add `do-not-care-about-stupid-american-laws' option to the [option](https://gitlab.freedesktop.org/mesa/mesa/-/blob/0c87909bcfeb2283d4326ef8a411598872802486/meson_options.txt#L541).

Or adding a `passthru.tests` script checking that the value we are passing includes all the listed options similarly to what we do e.g. in [fwupd](https://github.com/NixOS/nixpkgs/blob/e6896c17ca6d7897683eff69290b4dc1eb58a8b5/pkgs/os-specific/linux/firmware/fwupd/default.nix#L363-L383).

Reply:
I think I can write the test and add it in a future PR, but this I'll have to ask right now:
- Let's move it to a boolean, then?

Reply:
Sounds OK to me.